packages/components/src/Payments/PaymentsInputValidationUI.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import { colors, fonts, mediaQueries, ErrorIcon } from "../";
+import styled from "styled-components";
+
+export enum INPUT_STATE {
+  EMPTY = "empty",
+  VALID = "valid",
+  INVALID = "invalid",
+}
+
+export interface InputValidationStyleProps {
+  inputState?: string;
+  width?: string;
+}
+
+const InputWrapper = styled.div`
+  margin-bottom: 20px;
+  position: relative;
+  width: ${(props: InputValidationStyleProps) => (props.width ? props.width : "100%")};
+
+  ${mediaQueries.MOBILE} {
+    width: 100%;
+  }
+
+  input {
+    border: 1px solid
+      ${(props: InputValidationStyleProps) =>
+        props.inputState === INPUT_STATE.INVALID ? colors.accent.CIVIL_RED : colors.accent.CIVIL_GRAY_3};
+    border-radius: 2px;
+    padding: 10px 35px 10px 12px;
+    width: 100%;
+
+    &:focus {
+      border-color: ${colors.accent.CIVIL_BLUE};
+      outline: none;
+    }
+
+    ${mediaQueries.MOBILE} {
+      width: 100%;
+    }
+  }
+
+  select {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
```

Comment:
Just checked and according to https://www.styled-components.com/docs/basics styled-components handles vendor prefixes for us :pray: